osu.Framework/Platform/GameHost.cs

```diff
@@ -153,6 +164,14 @@ public abstract class GameHost : IIpcHost, IDisposable
 
         private ReadableKeyCombinationProvider readableKeyCombinationProvider;
 
+        protected Bindable<ScreenOrientation> ScreenOrientationBindable { get; private set; }
+
+        [CanBeNull]
+        protected virtual ScreenOrientationManager GetScreenOrientationManager() => null;
```

Comment:
```suggestion
        protected virtual ScreenOrientationManager CreateScreenOrientationManager() => null;
```

Should be `Create...` for classes that are single-instanced. To match `CreateWindow()`, `CreateReadableKeyCombinationProvider()`, etc.

`GetClipboard()` is creating a new clipboard every time, so it's `Get...`.